participants/yonus.json

```
@@ -0,0 +1,44 @@
+{
+	// your real name (required by location host)
+	"realName": {
+		"givenName": "Yonus",
+		"familyName": "Bakhshi",
+		// if you prefer to have your family name shown first (optional)
+		"placeFamilyNameFirst": false,
+		// if you do not want to show your family name on the participant list (optional)
+		"hideFamilyNameOnWebsite": false
+	},
+	// please put in the account name of the PR creator, if you sign up somebody else
+	"githubAccountName": "ybakhshi",
+	// company name (optional)
+	"company": "",
+	// either both days or at least one day has to be set to true.
+	// If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+	"when": {
+		// June 27th, 2025
+		"friday": false,
+		// June 28th, 2025
+		"saturday": true
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": true,
+	// your current interests (JS and in general) (required)
+	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
```

From the `whatIsMyConnectionToJavaScript`, it sounds like these could fit you:

```suggestion
	"tags": ["React", "Node", "JS Engines"],
```